package.json

```diff
@@ -18,11 +18,11 @@
     "@opentelemetry/sdk-trace-base": "^1.4.0",
     "node-addon-api": "^3.1.0",
     "node-gyp": "^9.0.0",
+    "npm": "^7.18.1",
```

Comment:
wait why is npm itself a dependency here? I see this directly after hitting approve.

Can you remove it?